app/components/home/stacking-card.tsx

```diff
@@ -1,68 +1,112 @@
 import React, { FC } from 'react';
 import { useSelector } from 'react-redux';
-import { Flex, Box, Text, color } from '@stacks/ui';
+import { Flex, Text, color } from '@stacks/ui';
 import BigNumber from 'bignumber.js';
 
 import { RootState } from '@store/index';
 import { formatPoxAddressToNetwork } from '@utils/stacking';
 import { selectStackerInfo } from '@store/stacking/stacking.reducer';
-import { PercentageCircle } from '@components/chart/percentage-circle';
-import { ExplainerTooltip } from '@components/tooltip';
+// import { PercentageCircle } from '@components/chart/percentage-circle';
+// import { ExplainerTooltip } from '@components/tooltip';
+import stackingImg from '@assets/images/abstract-btc-stx-bar-chart.svg';
+import {
+  InfoCard,
+  InfoCardGroup,
+  InfoCardLabel as Label,
+  InfoCardRow as Row,
+  InfoCardSection as Section,
+  InfoCardValue as Value,
+} from '@components/info-card';
+import { Hr } from '@components/hr';
+import { toHumanReadableStx } from '@utils/unit-convert';
+import { truncateMiddle } from '@utils/tx-utils';
+import { ExternalLink } from '@components/external-link';
 
-type StackingCardProps = any;
-
-export const StackingCard: FC<StackingCardProps> = () => {
+export const StackingCard: FC = () => {
   const { stackerInfo } = useSelector((state: RootState) => ({
     stackerInfo: selectStackerInfo(state),
   }));
 
+  if (!stackerInfo?.isCurrentlyStacking) return null;
+
   return (
-    <Flex
-      flexDirection="column"
-      mt="extra-loose"
-      borderRadius="8px"
-      boxShadow="0px 1px 2px rgba(0, 0, 0, 0.04);"
-      border={`1px solid ${color('border')}`}
-      px="loose"
-      minHeight="180px"
-    >
-      {stackerInfo?.isCurrentlyStacking && (
-        <Box>
-          <Flex mt="loose" justifyContent="center">
-            <PercentageCircle percentage={stackerInfo.stackingPercentage} />
-          </Flex>
-          <Text
-            display="block"
-            color={color('text-caption')}
-            textStyle="caption"
-            mt="base-tight"
-            textAlign="center"
-          >
-            Stacking progress
-          </Text>
-          <Flex justifyContent="center" mt="tight">
-            <Text textStyle="body.large.medium" fontSize="24px">
+    <InfoCard mt="extra-loose" px="extra-loose" minHeight="180px">
```

Comment:
Just curious why we settled on `loose`, `extra-loose`, etc as a convention instead of `sm`, `md`, `lg`, etc?

Reply:
Originates from https://polaris.shopify.com/design/spacing#navigation 

One issue with tshirt sizes is the temptation to have l, xl, xxl, xxxl etc